packages/panels/src/components/drawer/Drawer.tsx

```diff
@@ -0,0 +1,89 @@
+import styled from "@emotion/styled";
+import * as React from "react";
+import * as ReactDOM from "react-dom";
+
+export type SlideFrom = "left" | "right";
+
+export interface DrawerProps {
+  /** Ref to use for drawer. */
+  innerRef?: React.Ref<HTMLDivElement>;
+
+  portalTarget?: HTMLElement | null;
+  /* Whether the drawer is open or not. */
+  isOpen: boolean;
+  /**
+   * Which direction the drawer will appear from.
```

Comment:
Maybe specify options? We should probably do this everywhere, for people who aren't using Typescript. I missed this as well in my previous code :/